INSTALLATION.md

```diff
@@ -380,3 +390,106 @@ If you don't want this hook to run, you can manually opt out of this using the `
 
 <br/>
 ```
+
+# Talawa-Admin Service Setup Guide
+
+This guide outlines the steps to set up and manage the Talawa-Admin service on a Linux server using `systemd`.
+
+## Prerequisites
+
+- Firstly, You should have locally setup the Talawa-Admin repo using [Setting up this repository](#setting-up-this-repository)
+- Ensure **Node.js** and **npm** are correctly installed and available for the specified user and group.
+- It’s recommended to use **nvm** (Node Version Manager) for better management of different Node.js versions.
+- Ensure you have root or sudo access to configure systemd services.
+- Create a dedicated service user:
+   ```bash
+   sudo useradd -r -s /bin/false talawa_admin
+   sudo groupadd -r talawa_admin
+   sudo usermod -a -G talawa_admin talawa_admin
+   # Get the absolute path to your talawa-admin installation
+   INSTALL_PATH=$(pwd)
+   
+   sudo chown -R talawa_admin:talawa_admin $INSTALL_PATH
+   sudo chmod 750 $INSTALL_PATH
+   sudo find $INSTALL_PATH -type f -exec chmod 640 {} \;
+   sudo chmod 600 $INSTALL_PATH/.env
+   ```
+
+---
+
+### Service Configuration
+
+#### 1. **Copy the `talawa_admin.service` file**
```

Comment:
Make all these steps from 1 - 5 a numbered list under `Service Configuration`.  It's mostly a checklist.